ext/standard/hrtime.h

```diff
@@ -0,0 +1,58 @@
+/*
+   +----------------------------------------------------------------------+
+   | PHP Version 7                                                        |
+   +----------------------------------------------------------------------+
+   | Copyright (c) 1997-2017 The PHP Group                                |
+   +----------------------------------------------------------------------+
+   | This source file is subject to version 3.01 of the PHP license,      |
+   | that is bundled with this package in the file LICENSE, and is        |
+   | available through the world-wide-web at the following url:           |
+   | http://www.php.net/license/3_01.txt                                  |
+   | If you did not receive a copy of the PHP license and are unable to   |
+   | obtain it through the world-wide-web, please send a note to          |
+   | license@php.net so we can mail you a copy immediately.               |
+   +----------------------------------------------------------------------+
+   | Author: Niklas Keller <kelunik@php.net>                              |
+   +----------------------------------------------------------------------+
+*/
+
+/* $Id$ */
+
+#ifndef HRTIME_H
+#define HRTIME_H
+
+#define PHP_HRTIME_PLATFORM_POSIX   0
```

Comment:
Can migrate this, yep. In general, a system where sysconf is available could do a compile time check. I've omited it, because a single compile time failure is less to say against the runtime check. The binary can be compiled with a runtime check at the PHP startup, the submodule will fail otherwise.